src/main/java/com/vegaone/venus/domain/ProjectEntity.java

```diff
@@ -19,5 +19,7 @@ public class ProjectEntity {
     @JoinColumn(name = "company_id")
     @JsonIgnore
     private CompanyEntity company;
+    @OneToOne(mappedBy = "project")
```

Comment:
Here we need a `@ManyToMany` adnotation. One user can be assigned to many projects. 
So, the project will have a list of users and the user will have a list of projects.